.github/workflows/update-docker-mainnet.yml

```diff
@@ -0,0 +1,59 @@
+name: Update Mainnet Docker Image
+# This workflow is specifically designed to merge from main to mainnet
+# and publish the Docker image with the mainnet tag.
+
+# Due to EigenSDK package differences on Ethereum mainnet,
+# this workflow ensures the mainnet branch stays in sync with main
+# while maintaining its own Docker image tag.
```

Comment:
The original `mainnet.yml` is deleted and triggered manually by `update-docker-mainnet.yml` on-demand instead.